### PR TITLE
Added CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by Derek52**
#50 
I have written a CONTRIBUTING.md file.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+# Contribution Guide
+
+Firstly, thank you for considering making a contribution to our project! It's people like you that make Operation Code such a great community.
+
+We're an open source project and we love receiving contributions from our community - you! There are many ways to contribute to our projects, from writing tutorials or blog posts, improving documentation, submitting bug reports or feature requests, or writing code which can be merged into any of our repositories.
+
+The team at Operation Code urges all contributors to join our Slack team. Participating in discussions with the community on our Slack channel, is the best way to run new ideas by the team, and is the best place to get help. You can get an invitation to our Slack channel by [requestion to join Operation Code](https://operationcode.org/join). Once in our Slack team, simply type: '/open #oc-projects' and then click enter. Feel free to ask for help; everyone is a beginner at first :smile_cat:!
```
Comment (aaron-suarez) [inline CONTRIBUTING.md]: The channel associated with this particular project is actually #oc-python-projects. Can you please change this to read `Once in our Slack team, simply type: '/open #oc-python-projects'`?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+To work on the codebase for this project, you will need to have those dependencies installed.
+
+
+### Finding An Issue
+
+<details>
+	<summary>Click to Expand</summary>
+<ul>
+<li> After installing the listed dependencies, you can get to work coding on this project. A listing of this repo's can be found <a href="https://github.com/OperationCode/resources_api/issues">here</a></li>
+<li> Go to the issues page for this repo, [here]() and browse for an issue that you would like to work on. Don't be afraid to ask for help or clarification.</li>
```
Comment (aaron-suarez) [inline CONTRIBUTING.md]: I think these two items basically say the same thing. Also, there's a broken link in the second list item. I think we should combine them. Perhaps something like:
```html
<li> After installing the listed dependencies, you can get to work coding on this project. A listing of this repo's issues can be found <a href="https://github.com/OperationCode/resources_api/issues">here</a>. Browse for an issue that you would like to work on. Don't be afraid to ask for help or clarification.</li>
```

**Comment by Derek52** [comment]
I have made the changes to both the Readme and the contributing guide. I've committed them and pushed them to github. I'm not sure if I need to submit another pull request or not though. I believe you have the changes now. My edits should fix #57 and #59

**Comment by aaron-suarez** [comment]
Ok, so the general workflow is one issue per PR. It looks like you did two PRs on the same branch, so now they both have both changes.

I would like to see one PR for #57 and one PR for #59, which you've already started. Just take out the changes from the incorrect one in both of them. Does that make sense?

If you need some help with actually doing it, I'll be glad to walk you through it. Join our slack team (if you haven't already) and hit me up in #oc-python-projects and I'll help you fix it.

**Comment by aaron-suarez** [comment]
https://operationcode.org/join (in case you need an invite to slack)

**Comment by Derek52** [comment]
I can do that this evening. I think I'm going to just create 2 new branches to make sure I separate the issues. Can you just go ahead and close both of my open PR's? I'll submit 2 new PR's from my new branches.

**Comment by aaron-suarez** [comment]
Alright, sounds great. Thanks for understanding

**Pull request closed without merging**